readme.md

```diff
@@ -33,6 +33,7 @@ _NB: Beyond session 1, these sessions aren't in any particular order. There's so
     - [In the Name of Love](https://www.jacobinmag.com/2014/01/in-the-name-of-love/). Miya Tokumitsu.
     - ["Sharing" Economy and Self-Exploitation](http://thenewinquiry.com/blogs/marginal-utility/sharing-economy-and-self-exploitation/). Rob Horning.
     - [Racism and Science Fiction](http://www.nyrsf.com/racism-and-science-fiction-.html). Samuel R. Delany.
+    - [Starship Stormtroopers](http://flag.blackened.net/liberty/moorcock.html). Michael Moorcock.
```

Comment:
Essay by Michael Moorcock from the late 70s on how so much "utopian" SF writing is actually reactionary.